src/test/kotlin/org/dashevo/dapiclient/DapiGrpcClientTest.kt

```diff
@@ -67,7 +65,11 @@ class DapiGrpcClientTest {
         val client = DapiClient(nodes)
         try {
             //devnet-mobile, devnet genesis block
-            val block1 = PARAMS.devNetGenesisBlock
+            val block1 = when (PARAMS) {
+                is DevNetParams -> (PARAMS as DevNetParams).devNetGenesisBlock
+                is TestNet3Params -> Block(PARAMS, Utils.HEX.decode("020000002cbcf83b62913d56f605c0e581a48872839428c92e5eb76cd7ad94bcaf0b00007f11dcce14075520e8f74cc4ddf092b4e26ebd23b8d8665a1ae5bfc41b58fdb4c3a95e53ffff0f1ef37a00000101000000010000000000000000000000000000000000000000000000000000000000000000ffffffff0a510101062f503253482fffffffff0100743ba40b0000002321020131f38ae3eb0714531dbfc3f45491b4131d1211e3777177636388bb5a74c3e4ac00000000"))
```

Comment:
Can we do the same that we do with the DevNet genesis block?
`(PARAMS as TestNet3Params ).testNetGenesisBlock`

Reply:
In this case, I want to query block #1 (the second block).  For a devnet this is the devNetGenesisBlock, which is uniquely defined for each devnet.  But Block #1 for Testnet is the data taken from the test.  `(PARAMS as TestNet3Params ).testNetGenesisBlock` would be block 0.